src/components/Drawer/Drawer.Title.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2024 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ReactElement, ReactNode, useMemo } from 'react'
+
+import { H4 } from '../Typography/HX/H4'
+
+
+export type DrawerTitle = ReactNode
+
+export type TitleProps = {
+  title: DrawerTitle,
+}
+
+export const Title = ({ title }: TitleProps): ReactElement => {
+  const ellipsis = useMemo(() => ({ rows: 1, tooltip: title }), [title])
```

Comment:
It's better to use memo in this case: https://react.dev/reference/react/memo. This element should be rerendered only when title changes

Reply:
you mean by doing such as:

```ts
export const Title = memo(({ title }: TitleProps): ReactElement => {
   ...
})
```

Doing so may require changes in the future if we add more props other than title to render other stuff and need partial re-renders within the Title component 🤔